wagtail/admin/templates/wagtailadmin/home/upgrade_notification.html

```diff
@@ -1,10 +1,12 @@
 {% load i18n wagtailcore_tags wagtailadmin_tags %}
 {% wagtail_version as current_version %}
-<div class="w-panel-upgrade w-hidden w-flex w-mb-[-2rem] sm:w-mb-0 w-gap-5 w-items-center w-pl-slim-header w-pr-5 sm:w-px-[3.5rem] w-py-5 w-text-text-context w-bg-surface-info-panel w-border-b w-border-transparent"
```

Comment:
I replaced `data-w-upgrade-hidden-class="w-hidden"` and `w-hidden` for the initial state with the `hidden` attribute, because Tailwind's VS Code extension warned me that `w-hidden` and `w-flex` sets the same CSS property.